lib/twitch-soundboard/static/css/soundboard.css

```diff
@@ -1,4 +1,92 @@
+.divider1 {
+  float: left;
+  width: 30%;
+  margin: 1em;
+  border: solid black 10px;
+}
+
+.title1 {
+  height: 40px;
+  font-size: 30px;
+  color: blue;
+}
+
 button#gift {
```

Comment:
so you apply the CSS style to the html tag 'button' itself, then you add the id to each button inthe PUG file (which I think we already do).